docs/source/contrib_docs/doc-new-build.md

```diff
@@ -0,0 +1,33 @@
+# Building automatically on ReadTheDocs
```

Comment:
Is it practical to push improvements to RTD's own docs, then?

https://github.com/rtfd/readthedocs.org/tree/master/docs